scanner/src/components/Common/Title/index.svelte

```diff
@@ -1,4 +1,8 @@
-<div class={`title ${$$props.class}`}>
+<script lang="ts">
+  export let noBottomMargin: boolean
```

Comment:
This name is low quality, and it is clearly the default state. Why do we want the default to be explicitly mentioned, and why do we not give it a more obvious name as to what it is used for, or move the margin in the scope that needs the margin when it is needed - i.e. outside the `<Title>`?

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/44e5eaad-6125-43fe-9f85-2dea204fc4ed)

"No bottom margin" tells me what it does, but not when I should use it, or what it is for. Is this a "Top heading" or "sub heading" or what, what is the *logic* for when to use it? I wouldn't know when to use it just based on this name.